build-system/eslint-rules/html-template.js

```diff
@@ -65,6 +65,38 @@ module.exports = function(context) {
           ' HEAD root elements. Please do so manually with' +
           ' document.createElement.');
     }
+
+    const invalids = invalidVoidTag(string);
+    if (invalids.length) {
+      const sourceCode = context.getSourceCode();
+      const {start} = template;
+
+      for (let i = 0; i < invalids.length; i++) {
+        const {tag, offset} = invalids[i];
+        context.report({
+          node: template,
+          loc: sourceCode.getLocFromIndex(start + offset),
+          message: `Invalid void tag "${tag}"`,
+        });
+      }
+    }
+  }
+
+  function invalidVoidTag(string) {
+    // Void tags are defined at
+    // https://html.spec.whatwg.org/multipage/syntax.html#void-elements
+    const invalid = /<(?!area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)([a-zA-Z-]+) [^>]+\/>/g;
```

Comment:
@jridgewell This regex doesn't match all invalid void tags. For example, it doesn't match `<div />` or `<div/>`
https://regexr.com/3ojnm

Reply:
I have a fix we can do after this is merged. There are no linting errors even with it.

Reply:
Build failed again, so I pushed the fix here.